docs/sources/flow/reference/components/otelcol.processor.transform.md

```diff
@@ -0,0 +1,575 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.processor.transform/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.processor.transform/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.processor.transform/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.transform/
+labels:
+  stage: experimental
+title: otelcol.processor.transform
+description: Learn about otelcol.processor.transform
+---
+
+# otelcol.processor.transform
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" version="<AGENT VERSION>" >}}
+
+`otelcol.processor.transform` accepts telemetry data from other `otelcol`
+components and modifies it using the [OpenTelemetry Transformation Language (OTTL)][OTTL].
+OTTL statements consist of [OTTL functions][], which act on paths.
+A path is a reference to a telemetry data such as:
+* Resource attributes.
+* Instrumentation scope name.
+* Span attributes.
+
+In addition to the [standard OTTL functions][OTTL functions], 
+there is also a set of metrics-only functions:
+* [convert_sum_to_gauge][]
+* [convert_gauge_to_sum][]
+* [convert_summary_count_val_to_sum][]
+* [convert_summary_sum_val_to_sum][]
+
+[OTTL][] statements can also contain constructs such as:
+* [Booleans][OTTL booleans] such as:
```

Comment:
I wonder if each bullet here should just be the link without the examples. there's a lot more context and the same examples linked.

Reply:
I think for the "boolean expressions" the line is helpful:

> [Boolean Expressions][OTTL boolean expressions] consisting of a `where` followed by one or more booleans

But for the other two bullet points I agree - I'll just leave them as links.